execution/execution_test.go

```diff
@@ -173,7 +173,7 @@ func TestSubsidyLockTime(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			trx := tx.NewSubsidyTx(tc.lockTime, ts.RandAccAddress(), 1000, "subsidy-test")
+			trx := tx.NewSubsidyTx(tc.lockTime, ts.RandAccAddress(), 1000, tx.WithMemo("subsidy-test"))
```

Comment:
```suggestion
			trx := tx.NewSubsidyTx(tc.lockTime, ts.RandAccAddress(), 1000)
```
One of the advantages of making "memo" optional is to prevent some duplicated declarations in tests. This helps keep the tests clean. 

An example here:
The `TestSubsidyLockTime` is checking the "Lock Time for the Subsidy Transactions." The test name itself is clear, so we can remove the memo and keep the test simpler.

Reply:
Good suggestion, will make them simpler, yes.